README.md

```diff
@@ -1,3 +1,36 @@
-# myDiary
-myDairy is an online journal that allows users to make daily journal entries.
-The site is published at https://sammuel09.github.io/myDiary/
+# Project Title
+myDiary is a note taking/journaling web application.
+
+## Getting Started
+In order to have a copy of this project, clone this project to your local repository. You should have NodeJs installed on your Local machine. To start, you install the dependencies using NPM: npm install. This command installs all the dependencies this project runs on. Start the application on your localhost using the command: npm start.
+
+## Prerequisites
+You need to have NodeJs installed on yourlocal machine. Install Node here. You also need to have a working knowledge of Javascript and ES6 syntax.
+
+### Frameworks Used
+ExpressJs. Download as an npm package here
+
+### Coding Style
+This project was built using Javascript ES6. So, to transpile the ES6 code to ES5, you need to install BabelJS an NPM package.
+
+## Testing
+To test, you need to install mocha, chai and chai-http. You also need to install babel-register as NPM package to transpile ES6 code on the fly for testing.
+
+Run the test with the NPM command: npm test
+
+## Front End Views
+This app is hosted on Github Pages. The Front End views are published at https://sammuel09.github.io/myDiary/
+
+## BackEnd API
+The Backend APIs are hosted on heroku at https://mydiary-sammie.herokuapp.com/
+
+## Features
+This application has the following features:
```

Comment:
list the features like this 

* create entry
* edit entry

Reply:
I have updated it.